vuepress/docs/next/tutorials/samples/install-standard-demo.md

```diff
@@ -102,71 +105,71 @@ The application includes six custom micro frontends which are described below.
 
 ![SeedCardNG.png](./images/SeedCardNG.png)
 
-- The Card NG MFE is an Angular widget that is similar to the Card widget above except for the choice of front end technology.
+- The Card NG MFE is an Angular widget that is similar to the Card widget above,except for the choice of frontend technology.
 - This MFE communicates with Transaction Table widget which is implemented in React.
 
 #### 3. Manage Users
 
+
+
+- The Manage Users MFE makes an API call to Entando Identity Management to fetch user information. When the user is logged into the app, the MFE is visible
+  from the dropdown under the username.
+- By default application users are not provisioned authorization to manage users in Entando Identity Management. This demonstrates role based access control for an MFE using Keycloak. To enable the Manage Users widget, login to Keycloak and assign the `view-users` and `manage-users` roles from realm-management client to the desired user.
```

Comment:
"granted" instead of "provisioned".  Side note, "Entando Identity Management" doesn't seem like an active term in our glossary/vocabulary. I assume that means Keycloak so maybe we drop that clause here. We might need to survey usage of that phrase...